pyuploadcare/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 from __future__ import unicode_literals
 
-__version__ = '2.1.0'
+__version__ = '3.0.0'
```

Comment:
Bumping to 3.0.0  with deprecation warning for 4.x.x. based on the changelog.  What is your suggestion?